package.json

```
@@ -0,0 +1,89 @@
+{
+  "name": "architect-plugin-dynamodb-local",
+  "description": "Architect plugin for a local dynamodb instance",
+  "version": "1.0.0",
+  "repository": {
+    "type": "git",
+    "url": "github:nasa-gcn/architect-plugin-dynamodb-local"
+  },
+  "license": "Apache-2.0",
+  "contributors": [
+    {
+      "name": "Dakota Dutko",
+      "email": "dakota.c.dutko@nasa.gov"
+    }
+  ],
+  "type": "module",
+  "files": [
+    "index.js"
+  ],
+  "scripts": {
+    "prepare:husky": "husky",
+    "prepare:esbuild": "esbuild index.ts --bundle --packages=external --outfile=index.js --platform=node --format=esm --tree-shaking=true",
+    "prepare": "run-p prepare:*",
+    "test": "node --test"
+  },
+  "engines": {
+    "node": ">=20"
+  },
+  "dependencies": {
+    "@architect/functions": "^5.4.0",
+    "@aws-sdk/client-dynamodb": "^3.731.1",
+    "@aws-sdk/client-dynamodb-streams": "^3.731.1",
+    "@aws-sdk/client-ssm": "^3.731.1",
+    "@aws-sdk/lib-dynamodb": "^3.731.1",
+    "dedent": "^1.5.3",
+    "dockerode": "^4.0.0",
+    "env-paths": "^3.0.0",
+    "lodash.chunk": "^4.2.0",
```

The lodash package itself is better for tree-shaking.

```suggestion
    "lodash": "^4.2.0",
```